src/model_operations.py

```diff
@@ -52,21 +72,24 @@ async def generate_tokens(prompt, model):
         print(reason)
         yield reason
         return
-    
+
     try:
-        tokens = model.tokenize(prompt.encode())  # 'prompt.encode()' converts the string to bytes."
-    except Exception as e:
+        tokens = model.tokenize(
+            prompt.encode()
+        )  # 'prompt.encode()' converts the string to bytes."
+    except Exception:
```

Comment:
This change removed the exception (`e`), probably because it was not used. I would prefer to print the reason.

Reply:
Done